src/platform/packages/shared/kbn-esql-validation-autocomplete/scripts/generate_function_definitions.ts

```diff
@@ -959,6 +966,31 @@ ${
       });
       continue;
     }
+
+    if (arePredicates) {
```

Comment:
I will ask from ES to divide this definition from one file to two but for now we are handling this here